stack-packs/package.json

```diff
@@ -0,0 +1,8 @@
+{
+  "name": "@lighthouse/stack-packs",
```

Comment:
is this a scope we have?

Reply:
also need `"private": true,` (I think?) and `"license": "Apache-2.0",`

Reply:
private would be false as we will publish this to npm if it's in the other repo.

unsure about the scope @brendankenny just wanted it to look cool!

Reply:
I've send an email to npm support to request that org/scope. It appears to have been registered but it [remains unused](https://www.npmjs.com/org/lighthouse).

Reply:
ruh roh.  😢 
![image](https://user-images.githubusercontent.com/39191/55920219-e28ef400-5bad-11e9-8b53-747f4e389615.png)